nodetypes/http%3A%2F%2Fopentosca.org%2Fnodetypes/MariaDBMS_10-w2/NodeType.tosca

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:ns0="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="http://opentosca.org/nodetypes" id="nodeTypes-MariaDBMS_10-w2">
+    <NodeType name="MariaDBMS_10-w2" abstract="no" final="no" targetNamespace="http://opentosca.org/nodetypes" ns0:bordercolor="#00114f">
```

Comment:
Maybe add some inheritance to already existing types and use their defined properties?